.travis.yml

```diff
@@ -44,3 +48,19 @@ notifications:
 
 script:
   - make $MAKE_COMMAND -C package
+
+before_deploy:
+  - "mkdir -p upload"
+  - "mv package/*.tar.bz2 package/*.deb package/*.rpm package/*.zip upload || true"
+  - "rm -f upload/*source*"
+
+deploy:
+  provider: releases
+  api_key:
+    secure: fJMRmWSoBrm2hy6FDwhp7flHJhcXPZWE7gPluYT/G4K4uGf/QaFS0yKiTEXYEPWzBPUdidZxOKX427/rlmXyrl7ZfrUfn5k/cctc1aYpV5khtVWFCxlTbltzPefq8B7DkSDlQSnh9qB8e6J4E7xQd8zak6spjRntFBXafdvHM5E=
+  file: "upload/*"
+  file_glob: true
```

Comment:
Just upload everything in `upload`, I have tried to upload different files from different matrix jobs to the same tag, works.